mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -21,8 +21,6 @@ import java.io.PrintWriter
 
 import scala.reflect.ClassTag
 
```

Comment:
It's not in this story, but I'm curious why we don't exclude sharedParams in git, and dynamically call `SharedParamsCodeGen` in build script to ensure the consistency?